src/KeycloakGuardServiceProvider.php

```diff
@@ -8,9 +8,11 @@ class KeycloakGuardServiceProvider extends ServiceProvider
 {
   public function boot()
   {
-    $this->publishes([
-      __DIR__ . '/../config/keycloak.php' => config_path('keycloak.php'),
-    ], 'config');
+      $path = 'keycloak.php';
+
+      $this->publishes([
+          __DIR__ . '/../config/keycloak.php' => app()->basePath() . '/config' . ($path ? '/' . $path : $path),
```

Comment:
I don't get this ternary approach. 
`($path ? '/' . $path : $path),`

Once `$path = 'keycloak.php';`  the result always will be with slash `/keycloak`.